README.md

```diff
@@ -104,6 +104,8 @@ See code for all available configurations.
 | Lenovo ThinkPad L13               | `<nixos-hardware/lenovo/thinkpad/l13>`             |
 | Lenovo ThinkPad L14 (Intel)       | `<nixos-hardware/lenovo/thinkpad/l14/intel>        |
 | Lenovo ThinkPad L14 (AMD)         | `<nixos-hardware/lenovo/thinkpad/l14/amd>          |
+| Lenovo ThinkPad P1								| `<nixos-hardware/lenovo/thinkpad/p1/>`             |
```

Comment:
nitpick: This looks like tabs instead of spaces.

Reply:
Please also add the module to `flake.nix

Reply:
Did you forgot to push? I don't see the whitespace in the README and `flake.nix` is also unchanged.

Reply:
done